src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1179,6 +1180,282 @@ await using (var context = InMemoryApplicationDbContext(contextId))
+                TimePeriodCoverage = TimeIdentifier.TaxYear,
+                Publication = publication,
+                ReleaseName = "2035",
+                Slug = "2035",
+                PublishScheduled = DateTime.UtcNow,
+                Version = 0,
+                PreviousVersionId = initialReleaseId
+            };
+
+            var amendedRelease = new Release
+            {
+                Type = releaseType,
+                TimePeriodCoverage = TimeIdentifier.TaxYear,
+                Publication = publication,
+                ReleaseName = "2035",
+                Slug = "2035",
+                PublishScheduled = DateTime.UtcNow,
+                Version = 1,
+                PreviousVersionId = initialReleaseId
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.AddAsync(releaseType);
+                await context.AddRangeAsync(amendedRelease, initialRelease);
+                await context.SaveChangesAsync();
+            }
+
+            var releaseChecklistService = new Mock<IReleaseChecklistService>(MockBehavior.Strict);
+            var contentService = new Mock<IContentService>(MockBehavior.Strict);
+            var releaseFileService = new Mock<IReleaseFileService>(MockBehavior.Strict);
+
+            releaseChecklistService
+                .Setup(s =>
+                    s.GetErrors(It.Is<Release>(r => r.Id == amendedRelease.Id)))
+                .ReturnsAsync(
+                    new List<ReleaseChecklistIssue>()
+                );
+
+            contentService.Setup(mock =>
+                    mock.GetContentBlocks<HtmlBlock>(amendedRelease.Id))
+                .ReturnsAsync(new List<HtmlBlock>());
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var releaseService = BuildReleaseService(contentDbContext: context,
+                    releaseChecklistService: releaseChecklistService.Object,
+                    contentService: contentService.Object,
+                    releaseFileService: releaseFileService.Object);
+
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        amendedRelease.Id,
+                        new ReleaseStatusCreateViewModel
+                        {
+                            PublishScheduled = "2051-06-30",
+                            ApprovalStatus = ReleaseApprovalStatus.Approved,
+                            NotifySubscribers = true,
+                        }
+                    );
+
+                MockUtils.VerifyAllMocks(contentService, releaseFileService);
+
+                var viewModel = result.AssertRight();
+
+                Assert.True(viewModel.NotifySubscribers);
+            }
+        }
+
+        [Fact]
+        public async Task CreateReleaseStatus_Draft_NotifySubscribers()
```

Comment:
Feel like this one is already covered by `CreateReleaseStatus` which is testing the draft status update and you've added this assertion to that test already.